.github/workflows/regression-check.yml

```diff
@@ -15,6 +15,11 @@ on:
         description: '--diff-cutoff-percent'
         required: false
         default: 50
+  pull_request:
+    with:
+      benchmarks: ""
+      fields: "cputime allocated"
```

Comment:
`cputime` is not reliable especially on CI systems. It will fail all the time. We should use only allocations and the cutoff percent should be small like 3% because allocations usually remain exactly the same if there is no regression.